### PR TITLE
docs: Warn about the return code of the Main generated by /runAllTests:1

### PR DESCRIPTION
**Pull request by cpitclaudel**

### CODE REVIEW

File: Source/DafnyCore/DafnyOptions.cs [hunk 1]
```diff
@@ -1284,6 +1284,8 @@ verification outcome.
         be used if the program already contains a main method. Note that
         /compile:3 or 4 must be provided as well to actually execute
         this main method!
+        Beware: for now, the `Main` method generated by `/runAllTests` always
```
Comment (robin-aws) [inline Source/DafnyCore/DafnyOptions.cs:1287]:
```suggestion
        WARNING: for now, the `Main` method generated by `/runAllTests` always
```

Just turning up the volume a little :)

File: docs/DafnyRef/Attributes.md [hunk 2]
```diff
@@ -399,6 +399,8 @@ There are also two different approaches to executing all tests in a program:
    that invokes each test and prints the results.
    This runner is currently very basic, but avoids introducing any additional target
    language dependencies in the compiled code.
+   Beware: for now, the `Main` method generated by `/runAllTests` always returns
```
Comment (robin-aws) [inline docs/DafnyRef/Attributes.md:402]:
```suggestion
   WARNING: for now, the `Main` method generated by `/runAllTests` always returns
```

**Review by robin-aws — APPROVED** [review]

**Comment by robin-aws** [comment]
https://github.com/dafny-lang/dafny/pull/3047 has a proper fix for this, so if it looks good let's merge that instead!

**Comment by cpitclaudel** [comment]
Woohoo, closing this then :)

**Pull request closed without merging**